CONTRIBUTING.md

```diff
@@ -198,22 +198,29 @@ Add them for all new code, with the exception of:
 Make sure you import the types from `net.jcip`, there are homonyms in the classpath.
 
 
+### Nullability annotations
+
+We use the [Spotbugs annotations](https://spotbugs.github.io) to document nullability of parameters,
+method return types and class members.
+
+Please annotate any new class or interface with the appropriate annotations. Make sure you import 
+the types from `edu.umd.cs.findbugs.annotations`, there are homonyms in the classpath.
```

Comment:
👍 , it is unfortunately very easy to make that mistake.

Maybe we should also name the annotations explicitly, there is also a `@NotNull` in the classpath:
```
Please annotate any new class or interface with the appropriate annotations: `@NonNull`, `@Nullable`.
```